chat/command.go

```diff
@@ -155,7 +155,13 @@ func InitCommands(c *Commands) {
 			}
 
 			oldID := member.ID()
-			member.SetID(SanitizeName(args[0]))
+      newID := SanitizeName(args[0])
+      if newID == oldID {
+        body := "New nickname the same as previous nickname."
+        u.Send(message.NewSystemMsg(body, u))
```

Comment:
Done, I force pushed, not sure if you prefer new commits each time and then a squash at the end, but this is a pretty small change.